File: MySolution/front/src/models/Imc.ts
import { Aluno } from "./Aluno";

export interface Imc{
    criadoEm?: string
    imcId?: string;
    altura: number;
    peso: number; 
    imcTotal: number;
    classificacao: string;
    grauObesidade: string;
    aluno: Aluno;
    alunoId: string;
}

File: MySolution/API/Models/Imc.cs
namespace API.Models;

public class Imc
{
    public string? ImcId { get; set;} = Guid.NewGuid().ToString();
    public double? Altura { get; set; }
    public double? Peso { get; set; }
    public double? ImcTotal { get; set; }
    public string? Classificacao { get; set; }
    public string? GrauObesidade { get; set; }
    public Aluno? Aluno { get; set; }
    public string? AlunoId { get; set; }
}